Presubmit failures `FAILED tests/system/small/functions/test_remote_function.py::test_remote_function_unsupported_type` looks like a real one. I'll take a look.

--- a/tests/system/small/functions/test_remote_function.py
+++ b/tests/system/small/functions/test_remote_function.py
@@ -1646,7 +1646,7 @@ def test_remote_function_unsupported_type(
 
     with pytest.raises(
         ValueError,
-        match=r"'typing\.Sequence\[int\]' must be one of the supported types",
+        match=r"must be one of the supported types",
     ):
         bff.remote_function(
             input_types=int,

